### DISCUSSION TITLE
Flyby 0.13

### DISCUSSION THREAD

**Discussion by Belim** [builtbybel/FlyOOBE#6] [Announcements]
_Honestly, I wrote the tool in about an hour for a good friend, just to make a quick solution for a specific problem they were facing. Since it turned out to be quite useful, I thought I’d share it with the community as well. I'll drop the source code here later for anyone who's interested. In this way, others might benefit from it too, and who knows, it could be helpful to some people facing similar challenges. Also, we could probably expand on it in the future and add some more upgrades that are still on the way_

- The call and patch for the **native Inplace method of server setup** have been improved for better efficiency
- The **Mount and Run ISO** button, which previously only had a mounting function, can now inject the patch directly as soon as an ISO is available.
- Fixed a visual bug in the hover event. Descriptions now remain visible as long as the mouse click stays on an item https://github.com/builtbybel/Flyby11/issues/5#issuecomment-2438563790
- Added a new OOBE privacy panel that allows you to control your privacy settings. Use the option **Change how you experience Windows** for this
- If you prefer a fresh installation instead of an inplace upgrade, you can now do so via **I want to create Windows 11 installation media.** You can choose between the built-in options to create a bootable USB drive using Diskpart and DISM or even using Microsoft's Media Creation Tool. This method can write the bypass directly into the ISO via the autounattend.xml, similar to Rufus.  

<details><summary>More info on this feature can be found here</summary>

This update now also enables the creation of a bootable USB stick formatted as FAT32. For Windows installation media to boot from USB in UEFI mode, FAT32 formatting is required. However, if the install.wim file exceeds 4 GB, it can't be copied to a FAT32 drive. While the UEFI spec doesn't explicitly forbid booting from NTFS, it does require FAT32 boot support. For a UEFI/GPT Windows installation, the EFI boot partition on the target drive must be FAT32-formatted, as confirmed by Microsoft. Since Microsoft frequently increases the size of install.wim, eventually exceeding 4 GB, this prevents it from being copied to a FAT32 USB stick. To overcome this, I split the install.wim into two install.swm files, which are reassembled during setup.

I've also realized that UEFI systems expect the EFI System Partition (ESP), which is used for booting, to be in FAT32 format. That’s because UEFI firmware uses FAT32 to access boot files like bootx64.efi. If your USB stick is formatted in NTFS, your UEFI system might not be able to boot from it, as many UEFI systems don’t directly support NTFS without specific drivers or tweaks. That’s just a side note and doesn't need to concern you too much, but to sum it up, FAT32 booting is more reliable for me. The official Microsoft Media Creation Tool does it the same way. Throughout this process, I use only built-in Windows tools like DiskPart, Robocopy, and DISM without relying on external bootloaders like Rufus.
</details>

### Some of you want to thank me, so [I’ll drop my PayPal link here](https://paypal.com/donate?hosted_button_id=MY7HX4QLYR4KG). Thanks a lot, friends, and see you soon!

<hr /><em>This discussion was created from the release <a href='https://github.com/builtbybel/Flyby11/releases/tag/0.13'>Flyby 0.13</a>.</em>

**Comment by jalba1** [comment]
I tried this version and the anterior one without any succcess. I always obtain the following error message 
![Captura de pantalla 2024-10-27 211025](https://github.com/user-attachments/assets/90cee5ee-63dd-47dc-b1a7-b369df2aef70)

**Reply by jalba1** [reply]
I though that the problem could be solved using method 1, as described in the following comment: https://github.com/builtbybel/Flyby11/issues/5#issuecomment-2438563790. It seemd to everything were installing correctly but at the end the systems returned by itself to the anterior situation.
![Captura de pantalla 2024-10-28 144011](https://github.com/user-attachments/assets/369bc137-6da3-4349-a218-c57d9439fd01)
